src/MultiSelect/MultiSelect.svelte

```diff
@@ -22,11 +22,26 @@
   export let itemToString = (item) => item.text || item.id;
 
   /**
-   * Override the item name, title, labelText passed to the checkbox input
-   * @type {(item: MultiSelectItem) => { name?: string; labelText?: any; title?: string; }}
+   * Override the item name, title, labelText, or value passed to the user-selectable checkbox input as well as the hidden inputs.
+   * @type {(item: MultiSelectItem) => { name?: string; labelText?: any; title?:
```

Comment:
Nit: I think JSDoc parses this fine, but in my VSCode IDE the parsed `@type` is formatted strangely.

```suggestion
   * @type {(item: MultiSelectItem) => { name?: string; labelText?: any; title?: string; value?: string }}
```

<img width="529" alt="Screenshot 2023-10-25 at 10 10 12 PM" src="https://github.com/carbon-design-system/carbon-components-svelte/assets/10718366/df1a18dd-ce16-419a-9d02-a95a2016a890">